core-bundle/src/DependencyInjection/Configuration.php

```diff
@@ -184,8 +184,12 @@ private function addImageNode(): NodeDefinition
                             ->info('Allows to disable the layer flattening of animated images. Set this option to false to support animations. It has no effect with Gd as Imagine service.')
                         ->end()
                         ->scalarNode('interlace')
+                            ->info('One of the Imagine\Image\ImageInterface::INTERLACE_* constants.')
                             ->defaultValue(ImageInterface::INTERLACE_PLANE)
                         ->end()
+                        ->scalarNode('resampling-filter')
+                            ->info('Filter used when downsampling images. One of the Imagine\Image\ImageInterface::FILTER_* constants. It has no effect with Gd or SVG as Imagine service.')
+                        ->end()
```

Comment:
No `->defaultValue()` here?

Reply:
The contao/image library already uses a good default, so I would not add one here. Adding a default would change the hash for all images and that is not OK for a bugfix release IMO. If we want to add a default we would have to add it in Contao 5.5, but I don’t think we need that.